bbb-export-annotations/package-lock.json

```
@@ -19,6 +19,10 @@
       "devDependencies": {
         "eslint": "^8.20.0",
         "eslint-config-google": "^0.14.0"
+      },
+      "engines": {
+        "node": "^16.16.0",
+        "npm": "^8.11.0"
```

```suggestion
        "npm": "^8.5.0"
```

I missed this spot -- only changed it in package.json